app/views/shared/_header.html.erb

```diff
@@ -1,12 +1,13 @@
 <header class="headbar">
 	<div class='column centered'>
 		<div class="header-inner row centered">
-			<%= image_tag "logo.svg", class: 'logo', alt: 'Transit Talk Logo' %>
+			<a href="/">
+				<%= image_tag "logo.svg", class: 'logo', alt: 'Transit Talk Logo' %>
+			</a>
 			<a href="/">
 				<h1 class="title"><%= Setting.site_name %></h1>
 			</a>
 			<div class="row">
-				<div class= "search"></div>
```

Comment:
I think there's a lot of functionality related to this search bar. Let's leave it in

Reply:
Great! Committed.